docker-compose.yml

```diff
@@ -34,10 +34,11 @@ services:
       - minio
     volumes:
       - ./dev/faasm/build/:${FAASM_BUILD_MOUNT}
+      - ./dev/faasm-local/wasm/:/usr/local/faasm/wasm
```

Comment:
Not having both wasm and machine code mounted made it very confusing when they were out of sync.